script.js

```diff
@@ -22,3 +22,134 @@ function mobileMenu() {
 }
 
 mobileMenu();
+
+const cardPopUp = [
+  {
+    name: 'tonic',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    img: 'images/snapshoot/snapshoot_1.svg',
+    technologies: ['JavaScript', 'HTML', 'CSS'],
+    liveVersion: 'https://reem-lab.github.io/MyPortfolio/',
+    source: 'https://github.com/Reem-lab/MyPortfolio',
+  },
+
+  {
+    name: 'Multi-Post Stories',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    img: 'images/snapshoot/snapshoot_2.svg',
+    technologies: ['JavaScript', 'HTML', 'CSS'],
+    liveVersion: 'https://reem-lab.github.io/MyPortfolio/',
+    source: 'https://github.com/Reem-lab/MyPortfolio',
+  },
+  {
+    name: 'tonic2',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.2',
+    img: 'images/snapshoot/snapshoot_3.svg',
+    technologies: ['JavaScript', 'HTML', 'CSS'],
+    liveVersion: 'https://reem-lab.github.io/MyPortfolio/',
+    source: 'https://github.com/Reem-lab/MyPortfolio',
+  },
+  {
+    name: 'Multi-Post Stories2',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.2',
+    img: 'images/snapshoot/snapshoot_4.svg',
+    technologies: ['JavaScript', 'HTML', 'CSS'],
+    liveVersion: 'https://reem-lab.github.io/MyPortfolio/',
+    source: 'https://github.com/Reem-lab/MyPortfolio',
+  },
+];
+
+const ProjectElments = cardPopUp.map((ele, index) => {
+  const container = document.createElement('article');
+  if (index % 2 === 1) {
+    container.className = 'card flex flex-column bg row-reverse';
+  } else {
+    container.className = 'card flex flex-column bg';
+  }
+  container.innerHTML = ` <img src="${ele.img}" alt="project photo">
+    <div class="content">
+      <h3 class="heading bold cap">${ele.name}</h3>
+      <ul class="info flex">
+        <li class="category flex cross-center ellipse line-height highlight uppe">CANOPY</li>
+        <li class="category flex cross-center ellipse line-height highlight uppe">Back End Dev</li>
+        <li class="category flex cross-center ellipse line-height highlight uppe">2015</li>
+      </ul>
+      <p>
+      ${ele.description}
+      </p>
+      <ul class="tags flex">
+        <li class="line-height round-borders">${ele.technologies[0]}</li>
+        <li class="line-height round-borders">${ele.technologies[1]}</li>
+        <li class="line-height round-borders">${ele.technologies[2]}</li>
+      </ul>
```

Comment:
- [ ] Kindly make sure to follow the design and have matching padding and spacing around elements
#### EG
#### Your project(mobile view)
![image](https://user-images.githubusercontent.com/61377322/146305135-44f4c495-daf2-49b8-919b-5e3340736623.png)
#### Figma design(mobile view)
![image](https://user-images.githubusercontent.com/61377322/146305291-7524cf19-1aa4-432b-aba6-82958efef001.png)